GameData.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// Data of application.
+    /// </summary>
+    public class GameData
+    {
+        readonly public int[,] data;
+        readonly public int[,] newData;
```

Comment:
why do you need two arrays here?

Reply:
One is current iteration, one is the next iteration.

Reply:
you can always use only one array and when you are calculating the next iteration then you can just create one in the method and then give it values, no need to keep both accessible to the whole class.

And why are they readonly?

Reply:
Removed readonly and moved one array to inside a method.
As for why they were readonly, that was a mentor suggestion.